### DISCUSSION TITLE
"cannot get password" using vscode extension + solution

### DISCUSSION THREAD

**Discussion by reliablecontrols** [statelyai/xstate#3388] [General]
Hi All,
When I was trying to get setup, I got the following error message. xstate "cannot get password"

After a fair bit of problem solving (and cursing), I realized that the first time I was running vscode, I was running it as an admin (and logged into stately). When I restarted VS Code as a regular user, I didnt have permissions to logout (or login - since I was already logged in).  Re-openeing VSCode as admin, logging out of stately:, and restarting solved my problem.

Just sharing the solution in case anyone else gets caught.

**Comment by davidkpiano** [comment]
Thank you for sharing!
